src/SFML.System/ObjectBase.cs

```diff
@@ -78,5 +98,7 @@ private void Dispose(bool disposing)
         protected abstract void Destroy(bool disposing);
 
         private IntPtr myCPointer = IntPtr.Zero;
+
+        protected string MakeDisposedObjectString() => $"[{GetType().Name} (disposed)]";
```

Comment:
Can you add an documentation comment and I've move it above the `private` field

Reply:
Done.